packages/patternfly-react/src/components/Label/CompoundLabel.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Tooltip } from '../Tooltip';
+import { OverlayTrigger } from '../OverlayTrigger';
+import LabelWithTooltip from './LabelWithTooltip';
+
+class CompoundLabel extends React.Component {
+  generateTag = value => (
+    <LabelWithTooltip
+      key={value.id}
+      category={this.props.category}
+      value={value}
+      onDeleteClick={this.props.onDeleteClick}
+      truncate={this.props.valueTruncate}
+    />
+  );
+
+  render() {
+    const values = [...this.props.values];
+    if (values.length === 0) return <div />;
```

Comment:
return null would be better.

Reply:
any reason to return an empty div?

Reply:
No, sorry, I missed this comment, null will be better